pyproject.toml

```diff
@@ -27,6 +27,8 @@ langchain-nvidia-ai-endpoints = { version = "0.0.3", optional = true }
 colbert-ai = { version = "0.2.19", optional = true }
 pyarrow = { version = "14.0.1", optional = true }
 torch = { version = "2.2.1", optional = true }
+faiss-gpu = { version = ">=1.7.0", optional = true }
```

Comment:
Can we just lock on a version? Faiss can have compatibility issues if it's not tested.

Reply:
i just removed it from the dependency list